src/Kernel/ConstantBlockClosure.class.st

```diff
@@ -21,6 +23,21 @@ Class {
 	#category : #'Kernel-Methods'
 }
 
+{ #category : #'instance creation' }
+ConstantBlockClosure class >> numArgs: numArgs literal: aLiteral [
+	| classToUse |
+	classToUse := numArgs caseOf:
```

Comment:
It seems #caseOf message is going to be deprecated. Its protocol is "casing-To be deprecated".

Reply:
there are *many* users of caseOf. I tried to implement it without, but this was the nicest solution.